app/src/main/dist/config/crate.yml

```diff
@@ -478,18 +477,11 @@
 #discovery.zen.publish_timeout: 30s
 
 # Unicast discovery allows to explicitly control which nodes will be used
-# to discover the cluster. It can be used when multicast is not present,
-# or to restrict the cluster communication-wise.
-# For example, Amazon Web Services doesn't support multicast discovery.
-# Therefore, you need to specify the instances you want to connect to a
-# cluster as described in the following steps:
+# to discover the cluster. Therefore, you need to specify the instances you want
+# to connect to a cluster as described in the following steps:
 #
-# 1. Disable multicast discovery (enabled by default):
-#
-#discovery.zen.ping.multicast.enabled: false
-#
-# 2. Configure an initial list of master nodes in the cluster
-#    to perform discovery when new nodes (master or data) are started:
+# Configure an initial list of master nodes in the cluster
+# to perform discovery when new nodes (master or data) are started:
 #
 # If you want to debug the discovery process, you can set a logger in
```

Comment:
I think we should move this down below the list of nodes because there is a `:` at the end and then comes this sentence instead of the config section which is confusing... (could/should be done on master)